includes/reader-revenue/stripe/class-stripe-sync.php

```diff
@@ -221,7 +229,7 @@ private static function sync_to_esp( $customer, $args ) {
 					$subscription['start_date']
 				);
 				$metadata                   = array_merge( $recurring_related_metadata, $metadata );
-				if ( 'active' !== $subscription['status'] ) {
+				if ( ! in_array( $subscription['status'], [ 'active', 'trialing' ] ) ) {
```

Comment:
not sure why PHPCS didn't get this, but I we enforce the usage of strict comparison in `in_array`. So you need to add a third `true` argument there

Reply:
Thanks for flagging. Weird that neither CI nor local PHPCS caught it! I've fixed in https://github.com/Automattic/newspack-plugin/pull/2385/commits/07e465107c04a95ecdc8731fabec9e7103f0c3ee